app/models/game_session.rb

```diff
@@ -0,0 +1,30 @@
+class GameSession < ApplicationRecord
+  LIVES = 5
+
+  def lives
```

Comment:
I'd like to keep `LIVES` and `lives` as difference names, just to make it really clear to the reader about what they are.

Maybe `LIVES` could be `STARTING_LIVES`
and `lives` could be `lives_remaining`
?